readme/configuration-management.md

```diff
@@ -42,6 +52,45 @@ The solution is to make sure that the referenced content exists before the featu
 * Use the default_content module to export the referenced content as JSON files, and store these files with your feature or in a dependency.
 * Use Migrate and a custom module to create default content from any number of custom sources, such as JSON files stored with your feature.
 
+### Updating core and contributed modules
+
+Caution must be taken when updating core and contributed modules. If those updates make changes to a module’s configuration or schema, you must make sure to also update your exported configurations. Otherwise, the next time you run updates it will import a stale configuration schema and cause unexpected behavior.
+
+The best way to handle this is to always follow these steps when updating contributed and core modules:
+
+1. Start from a clean local setup or refresh. If you are using Features, ensure that there are no overridden configuration. The `cm.features.no-overrides` flag in [project.yml](https://github.com/acquia/blt/blob/8.x/template/blt/project.yml#L62) can assist with this by halting builds with overridden features.
+2. Use `composer update` to download the new module version(s).
+3. Run `drush updb` to apply any pending updates locally.
+4. If any updates were applied, check if they modified any stored configuration. If using Features, simply check for overridden features. If using core CM, re-export all configuration and check for any changes on disk.
+5. Re-export and commit any changes you found in the previous step, along with the updated `composer.json` and `composer.lock`.
+
+We need to find a better way of preventing this than manually monitoring module updates. Find more information in [these](https://www.drupal.org/node/2745685) [issues](https://github.com/acquia/blt/issues/842).
```

Comment:
This might just be me, but I'd rather have it formatted like:

> Find more information in these issues:
> 
>   - #XYZ: Linked issue 1 title
>   - #XYZ: Linked issue 2 title